interactions/api/models/channel.py

```diff
@@ -848,6 +846,11 @@ async def modify(
             payload["auto_archive_duration"] = auto_archive_duration
         if locked is not MISSING:
             payload["locked"] = locked
+        if position is MISSING:
+            if self.position is not None:
+                payload["position"] = self.position
```

Comment:
```suggestion
            if self.position is not None:
                payload["position"] = self.position
```
From what I see, the payload expects to get only the parameters which **need** to be modified. Therefore, why pass the current position, if we're not interested in changing it? Same logic appears for all parameters btw.

Reply:
That's a really good point

Reply:
Can you do the same for all non-necessary parameters?